app/components/provider_interface/reference_with_feedback_component.rb

```diff
@@ -53,6 +96,10 @@ def feedback_row
       end
     end
 
+    def safeguarding_feature_enabled?
```

Comment:
Could this method name better reflect the flag? (It's a bit odd to read it in the context of the referee relationship)

Reply:
@duncanjbrown [I've amended the last commit with a more accurate name](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1629/commits/74680880eccb246ae1df2e6b2dc4636684566be5#diff-0f5448dde4af0c77557a5e41679044a0R99).